src/RootStore/index.ts

```
@@ -62,6 +62,12 @@ if (process.env.NODE_ENV !== "test") {
     observableRequiresReaction: false,
   });
 
+  if (process.env.NODE_ENV !== "production") {
+    configure({
+      disableErrorBoundaries: true,
```

what is the need for this in, is it because of the state of the current data?

---

no it's just to aid with debugging in test and dev environments. without it Mobx swallows all the errors which generally just leads to mysterious UI behavior without a useful error message. It should only apply outside of deployment builds though